autoload/db_ui/schemas.vim

```diff
@@ -98,7 +98,7 @@ let s:oracle_args = 'echo "'.join(
       \    ";\n"
       \ ).';" |'
 let s:oracle_foreign_key_query = "
-      \ SELECT DISTINCT RFRD.table_name, RFRD.column_name, RFRD.owner
+      \SELECT DISTINCT RFRD.table_name, RFRD.column_name, RFRD.owner
```

Comment:
There were two spaces delimiting each query being run, we only need one space to delimit it so I changed it.